src/handlers/upload.ts

```diff
@@ -34,6 +35,9 @@ export const uploadHandler = async (
   res: Response,
   next: NextFunction
 ) => {
+  const { tags } = req.body;
+  const parsedTags = JSON.parse(tags);
```

Comment:
do you really need to `JSON.parse` it here?

Reply:
if so, you probably also want to validate that this is really just a `string[]`

Reply:
Yes, the parse is needed, because this is a request with content-type multipart/form-data.
How you suggest to make this validation for string[]?